doc/examples/segmentation/plot_hausdorff_distance.py

```diff
@@ -3,80 +3,70 @@
 Hausdorff Distance
 ==================
 
-This example shows how to calculate the Hausdorff distance between two sets of
-points. The `Hausdorff distance
+This example shows how to calculate the Hausdorff distance between a
+"ground truth" and a "predicted" segmentation mask. The `Hausdorff distance
 <https://en.wikipedia.org/wiki/Hausdorff_distance>`__ is the maximum distance
-between any point on the first set and its nearest point on the second set,
-and vice-versa.
+between any point on the contour of the first mask and its nearest point on the
+contour of the second mask, and vice-versa.
 
+It can be computed either directly from the segmentation masks using
+``hausdorff_distance_mask`` or from contours images (i.e. binary images where
+all the pixels on the contours are True) using ``hausdorff_distance``.
+
+In this example, the "contours image" is computed by removing the eroded mask
```

Comment:
```suggestion
In this example, the contour is computed by removing the eroded mask
```
Otherwise it sounds as if a "contours image" would be a *thing* (a type, cf. above).